doc/source/user-guide/vtk_to_pyvista.rst

```diff
@@ -94,13 +90,13 @@ Here, PyVista has done several things for us:
 #. The name for the :attr:`point_array <pyvista.point_array>` is given
    directly in dictionary-style fashion. Also, since VTK stores data
    on the heap (linear segments of RAM; a C++ concept), the
-   data must be flattened and put in Fortran ordering (which controls
-   how multidimensional data is laid out in physically 1d memory; numpy
-   uses "C"-style memory layout by default). This is why in our earlier
+   data must be flattened and put in FORTRAN ordering (which controls
```

Comment:
[According to the wiki](https://en.wikipedia.org/wiki/Fortran)

> The official language [standards](https://en.wikipedia.org/wiki/International_standard) for Fortran have referred to the language as "Fortran" with initial caps since Fortran 90.

Was this change here due to Google or due to ChatGPT? From the lack of changes I assume the former.

Reply:
Google. My fault for the all caps.